Editor/Window/ConnectToFleetInput.cs

```diff
@@ -120,15 +134,18 @@ private async Task UpdateFleetMenu()
         {
             if (_stateManager.GameLiftWrapper != null)
             {
-                var fleetList = await _fleetManager.ListFleetAttributes();
+                var fleetList = await _fleetManager.DescribeFleetAttributes();
                 if (fleetList == null)
                 {
-                    _fleetAttributes = new List<FleetAttributes>();
+                    var url = string.Format(Urls.AwsGameLiftLogs, _stateManager.Region);
+                    _connectToAnywhereStatusBox.Show(StatusBox.StatusBoxType.Error, Strings.AnywherePageStatusBoxDefaultErrorText, "", externalButtonLink: url, externalButtonText:Strings.ViewLogsStatusBoxUrlTextButton);
+                    _fleetAttributes = new List<FleetAttributes>(); 
                 }
                 _fleetNameDropdownContainer.choices = _fleetAttributes.Select(fleet => fleet.Name).ToList();
```

Comment:
I don't understand what's going on here. Where is `_fleetAttributes` being set? It is set to an empty list on line 29, so this will always be an empty list?

Reply:
This slipped through, removed the var and directly set `_fleetAttributes`